CMakeLists.txt

```
@@ -214,6 +214,18 @@ else()
   endif()
 endif()
 
+#-------------------------------------------------------------------------------
+# HIP Default Target Configuration.
+#
+# HIP does not have a stable instruction set like NVIDIA PTX; it requires
+# binaries specific to a target chip. We have tests that generate and run
+# deployable code which need to specify the proper target chip.
+#-------------------------------------------------------------------------------
+
+set(IREE_HIP_TEST_TARGET_CHIP \"\" CACHE STRING
+  "Target chip for HIP tests that need to compile device code. \
+   Defaults to \"\" to disable tests.")
```

not sure if this empty string code/comment format is correct... how about this?

```suggestion
set(IREE_HIP_TEST_TARGET_CHIP "" CACHE STRING
  "Target chip for HIP tests that need to compile device code. \
   Defaults to empty string to disable tests.")
```